test/middleware.js

```diff
@@ -616,5 +616,89 @@ describe('MongoTenant', function() {
         });
       });
     });
+
+    it('should bind tenant context to Model.updateMany().', function(done) {
+      let TestModel = utils.createTestModel({someField: String});
+
+      TestModel.create({tenantId: 'tenant1'}, {tenantId: 'tenant2'}, (err) => {
+        assert(!err, 'Expected creation of 2 test entities to work.');
+
+        TestModel.byTenant('tenant1').updateMany({}, {someField: 'some-value'}, (err) => {
+          assert(!err, 'Expected model updateMany to work.');
+
+          TestModel.byTenant('tenant1').find({}, (err, entities) => {
+            assert(!err, 'Expected entity search by Model.find to work.');
+
+            for (let entity of entities) {
+              assert.equal(entity.someField, 'some-value', 'Expected updated value of someField to be `some-value`.');
+            }
+
+            done();
+          });
+        });
+      });
+    });
+
+    it('should avoid overwriting tenant context on Model.updateMany().', function(done) {
+      let TestModel = utils.createTestModel({someField: String});
+
+      TestModel.create({tenantId: 'tenant1'}, {tenantId: 'tenant2'}, (err) => {
+        assert(!err, 'Expected creation of 2 test entities to work.');
+
+        TestModel.byTenant('tenant1').updateMany({}, {
+          tenantId: 'tenant2',
+          someField: 'some-value',
+          $set: { tenantId: "tenant2" }
+        }, (err) => {
+          assert(!err, 'Expected model updateMany to work.');
+
+          TestModel.byTenant('tenant1').find({}, (err, entities) => {
+            assert(!err, 'Expected entity search by Model.find to work.');
+            assert.equal(entities.length, 1, 'Expected to find exactly 1 entity.');
+            assert.equal(entities[0].someField, 'some-value', 'Expected updated value of someField to be `some-value`.');
+
+            done();
+          });
+        });
+      });
+    });
+
+    it('should preserve tenant context on Model.updateMany() with truthy overwrite option.', function(done) {
```

Comment:
:thinking: Correct me if i am wrong but in my understanding the `updateMany` api has no `overwrite` option at all.

Since you kept the console statement some lines below we can see an error that states ["multi update only works with $ operators"](https://travis-ci.org/github/craftup/node-mongo-tenant/jobs/695887375#L332). Since the overwrite option on eg. update or findOneAndReplace functions does exactly this, wrapping the update hash in "$set" command, it seems to underline my feeling.

Please be so kind and verify if this option really exists.